docs/articles/documentation/extending-testcafe/custom-reporter/creating-reporter-plugin/helper-methods.md

```diff
@@ -0,0 +1,297 @@
+---
+layout: docs
+title: Helper Methods
+permalink: /documentation/extending-testcafe/custom-reporter-plugin/helper-methods/
+---
+# Helper Methods
+
+The helper methods are used while [implementing a reporter](/testcafe/documentation/extending-testcafe/custom-reporter-plugin/creating-reporter-plugin/#implementing-reporter)
+to format a report output. TestCafe mixins these methods to the reporter.
+
+To refer to the methods, use `this`. The methods are chainable, which allows you to write compact code like this:
```

Comment:
Not all of them. All formatting helpers are not chainable, since they return strings